scripts/scripts.js

```diff
@@ -351,8 +352,17 @@ function buildCtaSections(main) {
     .forEach(buildCta);
 }
 
+function getDomainInfo(hostname) {
+  const domain = hostname.match(/^(?:.*?\.)?([a-zA-Z0-9\\_]{3,}\.(?:\w{2,8}|\w{2,4}\.\w{2,4}))$/);
```

Comment:
This needs to work with localhost as well, right? I tested localhost:3000 and got an error.

Reply:
will update the regex to match localhost as well

Reply:
updated